scipio/src/sys/uring.rs

```diff
@@ -210,12 +210,19 @@ where
         }
 
         let src = consume_source(value.user_data());
-        let source = mut_source(&src);
 
-        if try_process(source).is_none() {
-            let mut w = source.wakers.borrow_mut();
-            w.result = Some(value.result());
-            wakers.append(&mut w.waiters);
+        let result = value.result();
```

Comment:
can you fold this into the previous patch so we can keep everything as bisectable as possible?

Aside from that all good to go, thanks again!

Reply:
Done!